src/runtime/threading_backend.cc

```diff
@@ -164,9 +172,13 @@ class ThreadGroup::Impl {
       CPU_SET(id, &cpuset);
     }
 #if defined(__ANDROID__)
-    sched_setaffinity(thread, sizeof(cpu_set_t), &cpuset);
+    if (sched_setaffinity(tid, sizeof(cpu_set_t), &cpuset) == -1) {
+      LOG(WARNING) << "SetThreadAffinity fail!";
```

Comment:
possible to obtain more detail from strerror() or similar?

Reply:
Good advice.